main.cpp

```diff
@@ -416,7 +454,33 @@ Thing 7) Waiting Room
     2) give coffee to customers
     3) set the alarm
  */
-
+struct waitingRoom
+{
+  int numbOfManagers;
+  double monthlyBill;
+  std::string roomName;
+  bool lockedTable;
+  int coffeeMachines;
+
+  struct manager2
+  {
+    std::string managerName = "Saliha";
+    double monthlyFee = 4250.50;
+    bool isThisIntern = true;
+    int old = 23;
+    int dayOfWork = 5;
+  
+    double calculateDailySalary = monthlyFee / 30;
+    double checkTheBankAcconut;
+    void withdrawMoney(std::string accountName = "Personal", int accountNumber = 2);
+  };
+
+  void prepareToRec(int numberOfCabinet = 3, double startTime = 22.15);
+  void giveCoffee(std::string coffeType = "Arabica");
+  void setTheAlarm(bool enterPin = true, bool lockTheDoor = true);
+
+  manager2 jobDone;
+};
```

Comment:
Fix the following:
- Type Name capitalization
- Indentation relative to the `{ }`
- your nested class should have 3 member functions.  It only has 1: `withdrawMoney(...)`
- [ ] revise